server/blob/blob-s3/src/test/java/org/apache/james/blob/objectstorage/aws/DockerAwsS3Container.java

```diff
@@ -48,13 +54,12 @@ public DockerAwsS3Container() {
             .withEnv("LOG_LEVEL", "trace")
             .withEnv("REMOTE_MANAGEMENT_DISABLE", "1")
             .withCreateContainerCmdModifier(createContainerCmd -> createContainerCmd.withName("james-s3-test-" + UUID.randomUUID()))
-            .waitingFor(Wait.forLogMessage(".*\"message\":\"server started\".*\\n", ONE_TIME));
+            .waitingFor(Wait.forLogMessage(".*\"message\":\"server started\".*\\n", ONE_TIME))
+            .withStartupTimeout(STARTUP_TIMEOUT);
```

Comment:
**🎯 suggestion:** ‏having had to play with this container recently I wondered why the test container definition  doesn't include a log consumer ? the equivalent definitions for cassandra,  pulsar, postgres and some of the opensearch containers do. I can contribute the change but it could be easier to include it here :D
```suggestion
           `.withLogConsumer(DockerAwsS3Container::displayDockerLog);` 
            .waitingFor(Wait.forLogMessage(".*\"message\":\"server started\".*\\n", ONE_TIME))
            .withStartupTimeout(STARTUP_TIMEOUT);
```